### PR TITLE
Improved docs

### PR DESCRIPTION
**Pull request by comradekingu**

### CODE REVIEW

File: docs.phtml [hunk 1]
```diff
@@ -31,25 +31,25 @@
...
-    echo T_("Other resources are the <a href=\"https://wiki.gnucash.org/wiki/\">English</a> or <a href=\"https://wiki.gnucash.org/wiki/De/\">Deutsch</a> <span class=\"gnucash\">GnuCash</span> wikis.");
+    echo T_("Other resources are to be found in the <a href=\"https://wiki.gnucash.org/wiki/\">English</a> or <a href=\"https://wiki.gnucash.org/wiki/De/\">Deutsch</a> <span class=\"gnucash\">GnuCash</span> wikis.");
     ?></p>
 
   <p class="justify"><?php
-  echo T_("You can also send an email to the <a href=\"mailto:gnucash-user@gnucash.org\">gnucash-user</a> mailing list if you cannot find a satisfactory answer to your question within either the <b>Help Manual</b> or the <b>Tutorial and Concepts Guide</b>. We <b>want</b> feedback from you, it is also through your comments that we know how to improve the documentation.");
+  echo T_("Send an e-mail to the <a href=\"mailto:gnucash-user@gnucash.org\">gnucash-user</a> mailing list if you have tried the <b>Help Manual</b> and the <b>Tutorial and Concepts Guide</b>. Feedback and hints on what resources are missing is greatly appreciated.");
```
Comment (sunfish62) [inline docs.phtml:43]: Prefer (with appropriate tags):

Send an e-mail to the gnucash-user mailing list if you have tried the Help Manual and the Tutorial and Concepts Guide without finding an answer. Your contribution of additional feedback and hints is also greatly appreciated.

File: docs.phtml [hunk 2]
```diff
@@ -180,15 +180,15 @@
     </div>
   </div> <!-- stabledoc -->
 
-  <p class="justify"><?php echo T_("This is the latest stable release of the <span class=\"gnucash\">GnuCash</span> documentation. <em>Stable</em> means that this documentation is not necessarily the most up to date but it works without any problem.");?></p>
+  <p class="justify"><?php echo T_("This is the latest stable release of the <span class=\"gnucash\">GnuCash</span> documentation, meaning not necessarily up-to-date, but works without problems.");?></p>
```
Comment (sunfish62) [inline docs.phtml:183]: I prefer the original here.

File: docs.phtml [hunk 3]
```diff
@@ -483,7 +483,7 @@ $older7		="3";
 <div class="featureborder">
 <h2 class="featuretitle"><?php echo T_("Older GnuCash Documentation");?></h2>
 </div>
-<p class="justify"><?php echo T_("This section contains all the older <span class=\"gnucash\">GnuCash</span> documentation. If you are using one of these old versions of <span class=\"gnucash\">GnuCash</span>, it is highly recommended that you upgrade to the latest stable version.");?></p>
+<p class="justify"><?php echo T_("This section contains all the older <span class=\"gnucash\">GnuCash</span> documentation. If you are using one of these old versions of <span class=\"gnucash\">GnuCash</span>, it is highly recommended you upgrade to the latest stable one.");?></p>
```
Comment (sunfish62) [inline docs.phtml:486]: Prefer to leave "that" in the sentence, although "one" is an improvement.

File: docs.phtml [hunk 4]
```diff
@@ -712,7 +712,7 @@ $older7		="3";
 </ul>
 
 <div class="dialog note">
-<p class="justify"><?php echo T_("<b>NOTE:</b> the information presented in these sources may or may not be updated to the most recent version of <span class=\"gnucash\">GnuCash</span>. If you are interested in doing development work on these documents, you should contact the original authors.");?></p>
+<p class="justify"><?php echo T_("<b>Note:</b> Info presented in these sources may or may not be align with the latest <span class=\"gnucash\">GnuCash</span>. Contact the authors if you are interested in improving these resources.");?></p>
 </div>
```
Comment (sunfish62) [inline docs.phtml:716]: What about:

Note: The information presented in these resources may or may not be updated to the most recent version of GnuCash. Contact the original authors if you are interested in improving them.

File: docs.phtml [hunk 5]
```diff
@@ -31,25 +31,25 @@
...
+  printf (T_("Additionally, you can talk to someone by posting your question on IRC at %s."),
       sprintf ("<a href='%s'><i>irc.gnome.org</i> %s <b>#gnucash</b></a>", \URL_IRC,
           /* Translators: IRC channel */
           T_("channel")));
   ?></p>
 
   <div class="dialog note">
-    <p class="justify"><?php echo T_("<span class=\"gnucash\">GnuCash</span>'s documentation has been created by its community. See the <a href=\"develdocs.phtml\">Writing Documentation page</a> if you are interested in contributing to this effort.");?></p>
+    <p class="justify"><?php echo T_("The <span class=\"gnucash\">GnuCash</span> documentation is a community creation. The <a href=\"develdocs.phtml\">Writing Documentation page</a> helps you improve it.");?></p>
   </div>
```
Comment (sunfish62) [inline docs.phtml:53]: Prefer:

The GnuCash documentation is a community creation. The Writing Documentation page provides specific instruction and guidance on how you can contribute.

**Comment by sunfish62** [comment]
If we are going to change this page at all, I'd recommend removing the last section ("Unofficial Documentation") altogether. The documents referenced there date from 2001 and 2003, respectively. I'm pretty sure that GnuCash has changed a little bit since version 1.6. If someone really wants these resources available, bury them somewhere on the wiki.

File: docs.phtml [hunk 6]
```diff
@@ -8,11 +8,11 @@
    */
   include("lang.php");
   $title = T_("Documentation");
-  $description = T_("The following links are to the <strong>English HTML editions of the current stable branch</strong> documentation. See further down the page for other <span class=\"gnucash\">GnuCash</span> versions, languages, and formats.");
+  $description = T_("The following links go to the <strong>English HTML editions of the current stable branch</strong> documentation. Other <span class=\"gnucash\">GnuCash</span> versions, languages, and formats are below that.");
```
[the remark this answers is not in the document]
Comment (comradekingu) [reply docs.phtml:11]: I think the whole problem can be avoided.

Comment (comradekingu) [inline docs.phtml:11]:
```suggestion
  $description = T_("The <strong>English HTML documentation of the current stable branch</strong> is linked. Other <span class=\"gnucash\">GnuCash</span> versions, languages, and formats are at the bottom.");
```

File: docs.phtml [hunk 7]
```diff
@@ -31,25 +31,25 @@
     <li><a href="<?=$top_dir;?>/viewdoc.phtml?rev=<?=$major_stable;?>&amp;lang=C&amp;doc=guide"><b><?php echo T_("The Tutorial and Concepts Guide");?></b></a></li>
   </ul>
 
-  <p class="justify"><?php echo T_("The <b>Help Manual</b> is designed to be a quick reference of how to accomplish specific tasks and how to use the features in <span class=\"gnucash\">GnuCash</span>. The <b>Concepts Guide</b> is designed to be an in depth guide to the concepts behind using <span class=\"gnucash\">GnuCash</span> with a tutorial to show how to put those concepts into practice.");?></p>
+  <p class="justify"><?php echo T_("The <b>Help Manual</b> details specific tasks and how to use features <span class=\"gnucash\">GnuCash</span>. The <b>Concepts Guide</b> is an in-depth guide to <span class=\"gnucash\">GnuCash</span> tutorial.");?></p>
```
Comment (comradekingu) [inline docs.phtml:34]:
```suggestion
  <p class="justify"><?php echo T_("The <b>Help Manual</b> details specific tasks and features of <span class=\"gnucash\">GnuCash</span>, while the <b>Concepts Guide</b> is an in-depth guide.");?></p>
```

File: docs.phtml [hunk 8]
```diff
@@ -31,25 +31,25 @@
...
 
   <p class="justify"><?php
     /* Todo: menu.phtml has a similar link. They should be unified with the other language stuff.*/
     /* Translators: If wiki pages in your laguage exists, consider to replace "De"/"Deutsch" by your language.*/
-    echo T_("Other resources are the <a href=\"https://wiki.gnucash.org/wiki/\">English</a> or <a href=\"https://wiki.gnucash.org/wiki/De/\">Deutsch</a> <span class=\"gnucash\">GnuCash</span> wikis.");
+    echo T_("Other resources are to be found in the <a href=\"https://wiki.gnucash.org/wiki/\">English</a> or <a href=\"https://wiki.gnucash.org/wiki/De/\">Deutsch</a> <span class=\"gnucash\">GnuCash</span> wikis.");
```
Comment (comradekingu) [inline docs.phtml:39]:
```suggestion
    echo T_("Other resources available in the <a href=\"https://wiki.gnucash.org/wiki/\">English</a> or <a href=\"https://wiki.gnucash.org/wiki/De/\">Deutsch</a> <span class=\"gnucash\">GnuCash</span> wikis.");
```

File: docs.phtml (hunk 1 above)
Comment (comradekingu) [inline docs.phtml:43]:
```suggestion
  echo T_("Use the <a href=\"mailto:gnucash-user@gnucash.org\">gnucash-user</a> mailing list if the <b>Help Manual</b> and <b>Tutorial and Concepts Guide</b> doesn't solve your problem. Feedback, help and hints on what resources are missing is greatly appreciated.");
```

File: docs.phtml (hunk 5 above)
Comment (comradekingu) [reply docs.phtml:53]: Too long-winded. Keep it sniple snupple.

File: docs.phtml (hunk 2 above)
Comment (comradekingu) [reply docs.phtml:183]: I prefer not explaining this in an apologetic manner and rather just keep the documentation up-to-date. Just shorter is the best overall benefit to be had for now is what I think.

File: docs.phtml (hunk 3 above)
Comment (comradekingu) [inline docs.phtml:486]:
```suggestion
<p class="justify"><?php echo T_("This is the documentation for older <span class=\"gnucash\">GnuCash</span> versions. Upgrade if you can.");?></p>
```

Comment (comradekingu) [reply docs.phtml:486]: In my reading into why that is, I found the whole thing to be a challenge.

File: docs.phtml (hunk 4 above)
Comment (comradekingu) [reply docs.phtml:716]: It is from, respectively July 2001 and November 4, 2002. I think it is time to let go here.

File: docs.phtml [hunk 9]
```diff
@@ -698,23 +698,5 @@ $older7		="3";
 </ul>
 </div>
 
-<div class="featurecontainer">
```
Comment (comradekingu) [inline docs.phtml:701]: Guides were last updated respectively 2001 and 2002, and they can be reinstated by popular demand, but I think the biggest supporters will already have secured their copy by now.

File: docs.phtml (hunk 1 above)
Comment (jralls) [reply docs.phtml]: Prefer "email" to "e-mail".

File: docs.phtml [hunk 10]
```diff
@@ -8,11 +8,11 @@
    */
   include("lang.php");
   $title = T_("Documentation");
-  $description = T_("The following links are to the <strong>English HTML editions of the current stable branch</strong> documentation. See further down the page for other <span class=\"gnucash\">GnuCash</span> versions, languages, and formats.");
+  $description = T_("The <strong>English HTML documentation of the current stable branch</strong> is linked. Other <span class=\"gnucash\">GnuCash</span> versions, languages, and formats are at the bottom.");
```
Comment (jralls) [inline docs.phtml:11]: The original is idiomatic English, the change is not.

Comment (comradekingu) [inline docs.phtml:11]:
```suggestion
  $description = T_("<strong>HTML documentation of the current stable branch</strong> is provided in English. At the bottom of the page there are other <span class=\"gnucash\">GnuCash</span> versions, languages, and formats.");
```

File: docs.phtml (hunk 5 above)
Comment (jralls) [reply docs.phtml:53]: @comradekingu It may be longer but it is also idiomatic and correct. Your version is neither. @sunfish62 is a professional editor. You would be well advised to learn from him.

Comment (comradekingu) [reply docs.phtml:53]: In any professional capacity, I don't understand why referencing "it" is somehow worse than leaving "contribute" up in the air? I'll try to tie it together.

File: docs.phtml [hunk 11]
```diff
@@ -31,25 +31,25 @@
...
-  printf (T_("Additionally, you can talk to someone via IRC at %s about your question."),
+  printf (T_("Additionally, you can talk to someone by posting your question on IRC at %s."),
       sprintf ("<a href='%s'><i>irc.gnome.org</i> %s <b>#gnucash</b></a>", \URL_IRC,
           /* Translators: IRC channel */
           T_("channel")));
   ?></p>
 
   <div class="dialog note">
-    <p class="justify"><?php echo T_("<span class=\"gnucash\">GnuCash</span>'s documentation has been created by its community. See the <a href=\"develdocs.phtml\">Writing Documentation page</a> if you are interested in contributing to this effort.");?></p>
+    <p class="justify"><?php echo T_("The <span class=\"gnucash\">GnuCash</span> documentation is a community creation. The <a href=\"develdocs.phtml\">Writing Documentation page</a> helps you improve it.");?></p>
```
Comment (comradekingu) [inline docs.phtml:52]:
```suggestion
    <p class="justify"><?php echo T_("The <span class=\"gnucash\">GnuCash</span> documentation is a community effort you can improve with the help of the <a href=\"develdocs.phtml\">Writing Documentation page</a>.");?></p>
```

File: docs.phtml [hunk 12]
```diff
@@ -8,11 +8,11 @@
    */
   include("lang.php");
   $title = T_("Documentation");
-  $description = T_("The following links are to the <strong>English HTML editions of the current stable branch</strong> documentation. See further down the page for other <span class=\"gnucash\">GnuCash</span> versions, languages, and formats.");
+  $description = T_("<strong>HTML documentation of the current stable branch</strong> is provided in English. At the bottom of the page there are other <span class=\"gnucash\">GnuCash</span> versions, languages, and formats.");
```
Comment (comradekingu) [inline docs.phtml:11]:
```suggestion
  $description = T_("<strong>HTML documentation of the current stable branch</strong> is provided in English. At the bottom of the page you will find other <span class=\"gnucash\">GnuCash</span> versions, languages, and formats.");
```

File: docs.phtml (hunk 9 above)
Comment (fellen) [reply docs.phtml:701]: How about replacing the content by a link to https://wiki.gnucash.org/wiki/Documentation?

File: docs.phtml (hunk 5 above)
Comment (sunfish62) [reply docs.phtml:53]: Maybe "contribute" is up in the air for you, but it isn't for me. The difference between your version and mine is three words. And, for the record, your version suggests that a user's writing contributions would improve the community effort, rather than the actual documentation.

File: docs.phtml (hunk 2 above)
Comment (sunfish62) [reply docs.phtml:183]: How is your version "less apologetic"?!?!?!? If you're balking at the explanation of what "stable documentation" means, then remove the sentence altogether, and leave this as "This is the latest stable release of the GnuCash documentation."

However, I sincerely believe (after 15 years of using gnucash) that most regular users are unfamiliar with what "stable" means generally, and are unlikely to understand this project's particular interpretation of the term.

File: docs.phtml (hunk 9 above)
Comment (sunfish62) [reply docs.phtml:701]: Frank, the section contains two user guides from version 1.6. There is NO reason to include them on the website or the wiki. The content is both outdated and superceded.

File: docs.phtml [hunk 13]
```diff
@@ -31,25 +31,25 @@
     <li><a href="<?=$top_dir;?>/viewdoc.phtml?rev=<?=$major_stable;?>&amp;lang=C&amp;doc=guide"><b><?php echo T_("The Tutorial and Concepts Guide");?></b></a></li>
   </ul>
 
-  <p class="justify"><?php echo T_("The <b>Help Manual</b> is designed to be a quick reference of how to accomplish specific tasks and how to use the features in <span class=\"gnucash\">GnuCash</span>. The <b>Concepts Guide</b> is designed to be an in depth guide to the concepts behind using <span class=\"gnucash\">GnuCash</span> with a tutorial to show how to put those concepts into practice.");?></p>
+  <p class="justify"><?php echo T_("The <b>Help Manual</b> details specific tasks and features of <span class=\"gnucash\">GnuCash</span>, while the <b>Concepts Guide</b> is an in-depth guide.");?></p>
```
[the remark this answers is not in the document]
Comment (comradekingu) [reply docs.phtml:34]: It is actually the [GnuCash Tutorial and Concepts Guide](https://gnucash.org/viewdoc.phtml?rev=4&lang=C&doc=guide)
Could it be renamed just "Concepts"?

File: docs.phtml (hunk 12 above)
Comment (comradekingu) [reply docs.phtml:11]: Yes, however I think it starts off without any content, will revise.

File: docs.phtml (hunk 2 above)
Comment (comradekingu) [reply docs.phtml:183]: It isn't. Good idea. Agreed.

File: docs.phtml (hunk 9 above)
Comment (comradekingu) [reply docs.phtml:701]: There is a sidebar entry for the wiki, and "Documentation" is in both "Downloads" and "Support".
I think Download should be a button, and "Documentation" relegated to the "Support" section only.

**Comment by comradekingu** [comment]
I went ahead and did another pass trying to simplify matters into something everyone can (hopefully) understand.
Developers already know about stable/unstable waterfall rollout, and every other user is just needlessly confused by it.
Quickly running out of patience, and I don't get why everything has to be perfect for every pass. I'll see it again when I translate it.
Here it is in something akin to a final form, do with that what you will. Yes it is cumbersome to review things like that, I agree.

File: docs.phtml (hunk 7 above)
[the remark this answers is not in the document]
Comment (comradekingu) [reply docs.phtml]: Why am I not reading the documentation instead. I just want to get to it.

File: docs.phtml (hunk 1 above)
Comment (comradekingu) [reply docs.phtml]: e-mail has more errors in translation, and I'll for once champion the proper variant, even though I considered using it as a verb. :)

Comment (jralls) [reply docs.phtml]: Using "email" as a verb is common usage, so IMO "Email the gnucash-user mailing list" would be fine.

File: docs.phtml (hunk 13 above)
[the remark this answers is not in the document]
Comment (sunfish62) [reply docs.phtml]: If you're following official descriptive cataloging rules, then the Help Manual is "actually" the "GnuCash Help Manual" and go ahead changing that throughout the realm while you're at it.

The fact is, the document in question had been referred to as the Tutorial & Concept Guide for as long as I've known about it.

**Comment by sunfish62** [comment]
It would appear that your solution to constructive criticism is to simply ignore it all and put forth your own idea of what this page should say. Personally, I didn't see a huge problem with the way it was before-- but I considered your suggestions carefully and offered my own suggestions in response. As I am not the one who will approve your changes, my opinion and feelings are beside the point. However, I'd reject your PR if it were up to me.

**Comment by jralls** [comment]
> However, I'd reject your PR if it were up to me.

Concur. @fellen?

**Comment by comradekingu** [comment]
Honing the wording of an approach that isn't ultimately good doesn't make sense to me. Since we ended up doing that anyway, and there was no hope of merging this to be lost, I ended up trying to envision something that would make sense obsessing over. All of the good suggestions are taken into account, and the needlessly long style isn't there anymore. With this being translated now, it is a good time to do something about it.

I find this page has many qualities that can be reduced or omitted outright to the benefit of every user seeing it. So if doing a lot more than I had in mind for a first go, why should what it was decide in the matter of what it should be? That way it can be translated once, or in the very least translations end up being largely valuable rather than largely discarded. Translators like it when someone cares about the source strings, especially when it gets easier to translate resources.

There was a comment to the effect of stable/unstable not making sense as a concept to spring on users. Moreover the full matrix of options to that effect isn't even used, so I thought if I don't like it, and one person here thinks it hasn't even worked, why do it at all?

TL;DR I did see a problem, and now you can see something akin to what I would call a solution.

File: docs.phtml (hunk 13 above)
[the remark this answers is not in the document]
Comment (comradekingu) [reply docs.phtml]: There is no ampersand, and it is "concepts". It is referred to as  both "The Concepts Guide" and "The Tutorial and Concepts Guide" in https://www.gnucash.org/docs.phtml Those two variants stand out in bold, and they are not links, but instead the user is meant to know both of those refer to what was earlier described as how many parts it is divided into.

In the interest of the "we" in

> We want feedback from you, it is also through your comments that we know how to improve the documentation

> The Concepts Guide is designed to be an in depth guide to the concepts behind using GnuCash with a tutorial to show how to put those concepts into practice.

It says "to the concepts".
That doesn't bring anything beyond the title, which is already "Concepts", other than concepts having the quality of being "behind using GnuCash".
The concepts (double reference too), can be put into practice by way of "Concepts" the document being a guide.
I don't see a lot of helpful information beyond that.
There is a document called concepts, it is a guide, the guide is an in-depth one.

Why it being designed that way is important alludes me.
An in-depth guide amounting to a tutorial is a given, and maybe that is useful info somehow. If you don't know what a guide is maybe? I haven't read it, so I really don't know.
With that in mind, it is hard for me to get what is being communicated here.

It seems in a roundabout way an in-depth guide called Concepts is being presented as such, with a lot of non-supporting language to muddle things up.

From my cursory understanding of English and the GnuCash Concepts and Tutorial Guide, it isn't actually a guide(?)

[A tutorial](https://en.wikipedia.org/wiki/Tutorial), in education, is a method of transferring knowledge and may be used as a part of a learning process. More interactive and specific than a book or a lecture, a tutorial seeks to teach by example and supply the information to complete a certain task.

Edit: The help seems to be the same type of guide, covering some of the same topics, but with more info on specifics.
"Reports and Charts" says 

> This section is a “work in process”. Some of the material has not been reviewed for V2.0. While it may not be strictly accurate, it is at least a "guide".

That is at-least 10 years out of date (2.0), but maybe more like 15 (1.9.6).

**Comment by sunfish62** [comment]
So, you admit to ignoring the input of:

* long time contributors to gnucash code, 
* long time contributors to gnucash documentation, and
* native English speakers

All so that this page can be changed to suit your own idiosyncratic aesthetic on how this page should read. Nice!

File: docs.phtml (hunk 13 above)
[the remark this answers is not in the document]
Comment (sunfish62) [reply docs.phtml]: Wow. Talk about picky! I'm done with you. 

@fellen: this is why I dropped out of this community. I'll once again depart this loonie bin.

[the remark this answers is not in the document]
Comment (gjanssens) [reply docs.phtml]: > Wow. Talk about picky! I'm done with you.
> 
> @fellen: this is why I dropped out of this community. I'll once again depart this loonie bin.

And once again I'm sad to see this happen...

**Comment by gjanssens** [comment]
The conversation on this bug ended more than two years ago with no agreement on the proposed changes and no further attempts at aligning them with the feedback given. I don't see any use in keeping this PR open any longer.
